File: Python/Basic Maths/fibonacci.py
# program to print fibonacci series

num = int(input("Enter a number: "))
n1, n2, n3 = 0, 1, 0
print("Fibonacci Series is : ")
print("0 1 ", end="")
for x in range(2, num):
    n3 = n2 + n1
    print(n3, end=" ")
    n1 = n2
    n2 = n3

File: C++/Basic Maths/fibonacci.cpp
// program to print fibonacci series

#include <iostream>
using namespace std;

int main() {
    int num1 = 0, num2 = 1, n3, i, number;
    cout << "Enter a number :";
    cin >>  number;
    cout << "Fibonacci Series is :" << endl;
    cout << num1 << " ";
    cout << num2 << " ";
    for(i = 2; i < number; ++i) {
        n3 = num1 + num2;
        cout << n3 << " ";
        num1 = num2;
        num2 = n3;
    }
    return 0;
}